apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-tabs.tsx

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useMemo } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { RiPlayCircleLine } from 'react-icons/ri';
+import { useForm } from 'react-hook-form';
+// eslint-disable-next-line
+// @ts-ignore
+import { zodResolver } from '@hookform/resolvers/zod';
+import type { WorkflowTestDataResponseDto } from '@novu/shared';
+// import { TestWorkflowLogsSidebar } from './test-workflow-logs-sidebar';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '../../primitives/tabs';
+import { buildRoute, LEGACY_ROUTES, ROUTES } from '@/utils/routes';
+import { useFetchWorkflow } from '@/hooks';
+import { Form } from '../../primitives/form/form';
+import { Button } from '../../primitives/button';
+import { useTriggerWorkflow } from '@/hooks/use-trigger-workflow';
+import { showToast } from '../../primitives/sonner-helpers';
+import { ToastClose, ToastIcon } from '../../primitives/sonner';
+import { buildDynamicFormSchema, makeObjectFromSchema, TestWorkflowFormType } from '../schema';
+import { TestWorkflowForm } from './test-workflow-form';
+
+export const TestWorkflowTabs = ({ testData }: { testData: WorkflowTestDataResponseDto }) => {
+  const { environmentId = '', workflowId = '' } = useParams<{ environmentId: string; workflowId: string }>();
+  const { workflow } = useFetchWorkflow({
+    workflowId,
+  });
+  const to = useMemo(
+    () => (typeof testData.to === 'object' ? makeObjectFromSchema({ properties: testData.to.properties ?? {} }) : {}),
+    [testData]
+  );
+  const payload = useMemo(
+    () =>
+      typeof testData.payload === 'object'
+        ? makeObjectFromSchema({ properties: testData.payload.properties ?? {} })
+        : {},
+    [testData]
+  );
+  const form = useForm<TestWorkflowFormType>({
+    mode: 'onSubmit',
+    resolver: zodResolver(buildDynamicFormSchema({ to: testData?.to ?? {} })),
+    defaultValues: { to, payload: JSON.stringify(payload, null, 2) },
+  });
+  const { handleSubmit } = form;
+  const { triggerWorkflow } = useTriggerWorkflow();
+
+  const onSubmit = async (data: TestWorkflowFormType) => {
+    try {
+      const {
+        data: { transactionId },
+      } = await triggerWorkflow({ name: workflow?.workflowId ?? '', to: data.to, payload: data.payload });
+      showToast({
+        variant: 'lg',
+        children: ({ close }) => (
+          <>
+            <ToastIcon variant="success" />
+            <div className="flex flex-col gap-2">
+              <span className="font-medium">Test workflow triggered successfully</span>
+              <span className="text-foreground-600">{`Test workflow ${workflowId} was triggered successfully`}</span>
+              <Link
+                to={`${LEGACY_ROUTES.ACTIVITY_FEED}?transactionId=${transactionId}`}
```

Comment:
link to the old activity feed page with transactionId preselected so the user will be able to check the execution logs